cpp/src/slice2java/Gen.cpp

```diff
@@ -699,7 +699,7 @@ Slice::JavaVisitor::writeSyncIceInvokeMethods(
 
     // Generate a synchronous version of this operation which doesn't takes a context parameter.
     out << sp;
-    writeProxyDocComment(out, p, package, dc, false, "");
```

Comment:
Not essential to the PR, but this name was just bad.
It writes the doc-comment for a proxy operation, not the proxy itself.